### PR TITLE
Add a "go get" to the generate so it does not fail first time

### PR DESCRIPTION
**Pull request by asalkeld**

### CODE REVIEW

**Comment by asalkeld** [comment]
I'll vendor instead

File: Gopkg.toml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+required = ["github.com/jim-minter/go-cosmosdb/cmd/gencosmosdb", "github.com/alvaroloes/enumer"]
```
Comment (asalkeld) [inline Gopkg.toml:1]: this is needed as there is no import in the code and it keeps removing it

File: Gopkg.toml [hunk 2]
```diff
@@ -55,6 +57,14 @@
   branch = "release-4.3"
   source = "https://github.com/openshift/cluster-api-provider-azure"
 
+[[constraint]]
+  name = "github.com/jim-minter/go-cosmosdb"
+  revision = "323658c998227f194221880018cfd7a199fd714c"
```
Comment (jim-minter) [inline Gopkg.toml]: `branch = "master"` ?

**Review by jim-minter — CHANGES_REQUESTED** [review]
1 nit then will merge, thanks

File: Gopkg.toml (hunk 2 above)
Comment (asalkeld) [reply Gopkg.toml]: it doesn't seem to require it, but i thought you wanted to fix these generators? so ci didn't misbehave?

Comment (asalkeld) [reply Gopkg.toml]: so you can't specify master and revision "dep ensure" barfs
i thought the point was to nail down the version so we would have to update it?

Comment (asalkeld) [reply Gopkg.toml]: I am being a dummy, change coming..

**Comment by jim-minter** [comment]
rebased and merged manually, thanks

**Pull request closed without merging**